exercises/luhn/luhn_test.exs

```diff
@@ -0,0 +1,44 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("luhn.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule LuhnTest do
+  use ExUnit.Case
+
+  test "checksum" do
+    assert Luhn.checksum("4913") == 22
+  end
+
+  @tag :pending
+  test "checksum again" do
```

Comment:
No strong feelings regarding merging tests, just discussing possibilities. 👍 
 Nowadays I'm experimenting with this approach when the new test is not about a new scenario or edge case.